app/Commands/NewCommand.php

```diff
@@ -109,6 +114,11 @@ public function setConfig()
             return $this;
         });
 
```

Comment:
I feel like this part here where I'm reading and merging the saved configuration could be extracted into its own class.